### DISCUSSION TITLE
Series Tab and Custom Tags Request.

### DISCUSSION THREAD

**Discussion by CyberCimmerian** [openaudible/openaudible#926] [Ideas]
Hi.

I'm enjoying Open Audible and find it exceptionally good not just for downloading and file conversion, but also for organizing my library. After installing libation and Book Lib Connect for comparison, Open Audible is the only one that has that organization function. 

Having said that I also noticed that the other apps give you the option of adding custom tags (libation), and a Series tab that would be very much appreciated. i order to better organize your library.

Thanks.

**Comment by openaudible** [comment]
Thank you for the kind words and feedback.

We track the series... You can show all books in a series by clicking on the blue "Series" text on the Book Info. Clicking on the name of the series should take you to the Audible page about that series. 

Custom tags could be interesting-- but not sure what you want.. A command separated list of words that can be assigned to each book?

**Reply by CyberCimmerian** [reply]
Thanks for the reply.

In regards of the series, I meant to have a tab that has the Seies information always on display like so: (attached pic). That way you can get the info without going anywhere.

I'm thinking that the custom tags could be useful for personal tracking like for example a tag that says (TBR) To-Be-Read/listened, DNF (Did not finish), etc. Other tags that I can think of would be for example, tropes like Mary Sue, Daphne, Info-Dump, Unreliable Narrator.
![Captura de pantalla 2022-04-20 235625](https://user-images.githubusercontent.com/99073405/164376383-a6835061-0c45-4c48-89e4-8dba923692e9.png) 

Thank you for your time.

**Comment by openaudible** [comment]
Ah, a series book table column. Got it. 
Marking both as feature requests.
Thank you.

**Reply by ms162223** [reply]
Having a series column would be a huge help.  I have over 1200 books in my Audible library and it is so hard to sort and organize them by series.

**Comment by oboe45** [comment]
Definitely agree

**Comment by openaudible** [comment]
OK. I added it for a test release. It doesn't really add a lot of value in my opinion.. but I'll post a beta with it in there. 
You can add whatever you want to the "notes" section of a book. So having the option to display notes in the table might be nice too. 
Custom tags are probably outside of the scope of near term development.

**Comment by openaudible** [comment]
Added ability to reorder and add new tags to the book table. Also ability to delete aax files. And smaller preferences height.
https://openaudible.org/beta
